blank_project/package.json

```diff
@@ -1,16 +1,22 @@
 {
   "name": "@wasm/hello_world_ts",
   "description": "",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "scripts": {
-    "build": "node_modules/near-shell/near build",
-    "test": "jest test --env=./src/local_test_environment.js",
-    "test-on-devnet": "jest test --env=./src/devnet_test_environment.js"
+    "build": "mkdir -p out/ && gulp",
+    "deploy:contract": "near deploy --account_id=near-hello-devnet --node_url=https://studio.nearprotocol.com/devnet",
+    "deploy:pages": "gh-pages -d src",
+    "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
+    "prestart": "npm run build && npm run deploy:contract",
+    "start": "python3 -mhttp.server --directory src",
+    "test": "npm run build && jest test --env=near-shell/local_test_environment",
```

Comment:
I liked the test environment better under src, to not pollute the top level dir

Reply:
@janedegtiareva it's used from `near-shell` node module (i.e. it's not present in project template at all anymore).

Reply:
It's a good file to have in the project template to allow devs to modify the default behaviour.

Reply:
ah, I see, there is some app-specific stuff

It looks like files are largely identical though, except `nodeUrl`. So I think probably best would be to load this stuff from env-specific config and then have common test env use project-specific config.

Reply:
Some sort of base makes sense

Reply:
Leaving as is for this PR, gonna clean-up as part of https://github.com/nearprotocol/near-shell/issues/21